core-common/src/main/java/org/zalando/nakadi/validation/ISO4217CurrencyValidator.java

```diff
@@ -0,0 +1,40 @@
+package org.zalando.nakadi.validation;
+
+import org.everit.json.schema.FormatValidator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Currency;
+import java.util.Optional;
+
+public class ISO4217CurrencyValidator implements FormatValidator {
```

Comment:
lets be explicit that it is not currency validation, but rather currency code validation
```suggestion
public class ISO4217CurrencyCodeValidator implements FormatValidator {
```